<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvancedFilterModifyDialog</class>
 <widget class="QDialog" name="AdvancedFilterModifyDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>144</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New filter</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QFormLayout" name="formLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Filter name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSelectAllLineEdit" name="filterName">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>New filter</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Combine mode</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QRadioButton" name="radioAdd">
           <property name="toolTip">
            <string>Add result of this filter to the previous one.</string>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
           <property name="icon">
            <iconset resource="res/icons.qrc">
             <normaloff>:/ic_af_add.png</normaloff>:/ic_af_add.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioSubtract">
           <property name="toolTip">
            <string>Subtract result of this filter from previous one.</string>
           </property>
           <property name="text">
            <string>Subtract</string>
           </property>
           <property name="icon">
            <iconset resource="res/icons.qrc">
             <normaloff>:/ic_af_sub.png</normaloff>:/ic_af_sub.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioIntersect">
           <property name="toolTip">
            <string>Results that matches this filter and all previous will be present in the result.</string>
           </property>
           <property name="text">
            <string>Intersect</string>
           </property>
           <property name="icon">
            <iconset resource="res/icons.qrc">
             <normaloff>:/ic_af_intersect.png</normaloff>:/ic_af_intersect.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QSelectAllLineEdit</class>
   <extends>QLineEdit</extends>
   <header>widgets/QSelectAllLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>filterName</tabstop>
  <tabstop>radioAdd</tabstop>
  <tabstop>radioSubtract</tabstop>
  <tabstop>radioIntersect</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="res/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AdvancedFilterModifyDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AdvancedFilterModifyDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
